mpcs/src/lib.rs

```diff
@@ -357,14 +357,25 @@ fn err_too_many_variates(function: &str, upto: usize, got: usize) -> Error {
     })
 }
 
-#[cfg(test)]
+// TODO: Need to use some functions here in the integration benchmarks. But
+// unfortunately integration benchmarks do not compile the #[cfg(test)]
+// code. So remove the gate for the entire module, only gate the test
+// functions.
+// This is not the best way: the test utility functions should not be
+// compiled in the release build. Need a better solution.
+#[doc(hidden)]
```

Comment:
in zkevm-circuits, we have a "test" feature flag. and we write "#[cfg(any(feature = "test", test))]" a lot of places.

Reply:
That means we run benchmarks using `cargo bench --features "test"`?